### PR TITLE
Desktop Version

### PR DESCRIPTION
**Pull request by nouman17610**
I implemented the following areas in desktop version

- Header section
- Work section
- About section
- Contact form

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ After you're finished please remove all the comments and instructions!
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
```
Comment (Munsa1) [inline README.md]: - [ ] Good job creating a professional readme file, however, kindly remove **ALL**  comments that are not relevanto to the readme file for a more professional documentation.

image
image
image
image
image
image

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ After you're finished please remove all the comments and instructions!
...
   
 
-  <h3><b>Portfolio Mobile Screen ReadMe</b></h3>
+  <h3><b>Portfolio Website ReadMe</b></h3>
 
 </div>
```
Comment (Munsa1) [inline README.md:8]: - [ ] Kindly remove the broken links in your `table of contents` that lead the user to sections which do not exist.

e.g
- Kindly remove the `Live Demo` link as the readme file does not have a `Live Demo` section
image

- Kindly remove the `FAQ` section as well as the FAQ section does not exist as well
image

File: index.html [hunk 3]
```diff
@@ -9,99 +9,117 @@
...
+          <li>
+            <a href="#">Portfolio</a>
+          </li>
+          <li>
+            <a href="#">About</a>
+          </li>
+          <li>
+            <a href="#">Contact us</a>
+          </li>
+        </ul>
```
Comment (Munsa1) [inline index.html]: - [ ] Kindly note that it is a requirement that the implementation of the desktop version should not break the design of the mobile version. Kindly fix your mobile versionto match the given design

### Your design


Introductory section

image

Works Section

image

About Section

image


### Given layout

Inroduction

image

Works Section

image

About Section

image

image

File: index.html [hunk 4]
```diff
@@ -9,99 +9,117 @@
...
-      </p>
+        <div class="work-image">
+          <img src="images/work-1.png" alt="first-work">
+        </div>
+        <div class="details">
+          <div class="work-head">
+            <h2>Tonic</h2>
+          </div>
+          <div class="work-detail">
+            <p>CANOPY</p>
```
Comment (Munsa1) [inline index.html:96]: - [ ] Kindly ensure this tag has the same color and font-weight as the given design as this is a requirement

Your design
image

Given design
image

### Note this applies to all the cards

File: index.html [hunk 5]
```diff
@@ -9,99 +9,117 @@
...
+                  <li>
+                    <a heref="#"><img src="images/vector-icon.png" alt="social-media-icons">
+                    </a>
+                  </li>
+                </div>
+              </ul>
+        </div>
+      </div> 
     </section> 
     <section class="work-area">
```
Comment (Munsa1) [inline index.html:85]: - [ ] The second and last cards from the top in the `works` section have their card images to the right and the card description to the right. Kindly implement this as well

Given design
image

Your design
image

### Design for the last card from the top

Given design
image

Your design
image

### Note 🔴 
Kindly change your text to reflect the text in each card for a design that is more close to the Figma template

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @nouman17610, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Good UI Design :heavy_check_mark: 
Descriptive PR Description :heavy_check_mark: 
Good Commit Messages :heavy_check_mark: 
Media Queries Used Correctly ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 6]
```diff
@@ -315,12 +417,414 @@
   }
 }
 
-@media (min-width: 375px) {
+/** Desktop CSS Area **/
+
+@media (min-width: 768px) {
```
Comment (tugsem) [inline style.css:422]: -  [ ] Good job on fixing the layout of the mobile version as requested by the reviewer, however now your app on screens **768px** in width, seems like overlapping some text to each other. Kindly fix it for all of the cards and make sure to check screen sizes from 768px to upper and have the same layout.
image

**Review by tugsem — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️
Hello @nouman17610 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

 ## Highlights 🚀
✔️Good job on clearing readme and fixing the mobile version of the app
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @nouman17610, 
Approved 1

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Fixed Overlap As Requested By Previous Reviewer :heavy_check_mark: 
Achieved A Good Responsive Design :heavy_check_mark: 
Good Commit Messages :heavy_check_mark: 

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**